libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -324,7 +339,16 @@ internal class MapboxTripSession(
                 bannerInstructionEvent.latestBannerInstructions,
                 bannerInstructionEvent.latestInstructionIndex,
                 lastVoiceInstruction
-            )
+            ).also {
+                if (it == null) {
+                    logD(
+                        "route progress update dropped - " +
+                            "currentPrimaryRoute ID: ${primaryRoute?.id}; " +
```

Comment:
may be?
```suggestion
                            "current routes IDs: routes.map{ it.id} " +
```

Reply:
We don't have access to all routes here, only the primary one is kept locally.